src/library/sidebarmodel.cpp

```diff
@@ -610,3 +634,100 @@ void SidebarModel::slotFeatureSelect(LibraryFeature* pFeature,
     }
     emit selectIndex(ind, scrollTo);
 }
+
+void SidebarModel::saveCurrentSelection(const QModelIndex& index) {
+    if (!m_pConfig || !index.isValid()) {
+        return;
+    }
+
+    QString featureTitle;
+    QString childData;
+
+    if (index.internalPointer() == this) {
+        // root feature selected
+        featureTitle = m_sFeatures[index.row()]->title().toString();
```

Comment:
This stores the translated string of the feature. Not a problem in the first place, but it will break whenever users change the language or the translation changes between versions. The title is also used for progress and count information which changes during runtime. Can we use a programmatic string instead? Something like `name()`?